apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/kontrollpunkt/DoedshendelseKontrollpunktService.kt

```diff
@@ -0,0 +1,85 @@
+package no.nav.etterlatte.grunnlagsendring.doedshendelse.kontrollpunkt
+
+import kotlinx.coroutines.runBlocking
+import no.nav.etterlatte.common.klienter.PdlTjenesterKlient
+import no.nav.etterlatte.common.klienter.PesysKlient
+import no.nav.etterlatte.common.klienter.SakSammendragResponse.Companion.ALDER_SAKTYPE
+import no.nav.etterlatte.common.klienter.SakSammendragResponse.Companion.UFORE_SAKTYPE
+import no.nav.etterlatte.common.klienter.SakSammendragResponse.Status.LOPENDE
+import no.nav.etterlatte.common.klienter.SakSammendragResponse.Status.OPPRETTET
+import no.nav.etterlatte.common.klienter.SakSammendragResponse.Status.TIL_BEHANDLING
+import no.nav.etterlatte.grunnlagsendring.doedshendelse.Doedshendelse
+import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.libs.common.pdl.PersonDTO
+import no.nav.etterlatte.libs.common.person.PersonRolle
+
+class DoedshendelseKontrollpunktService(
+    private val pdlTjenesterKlient: PdlTjenesterKlient,
+    private val pesysKlient: PesysKlient,
+) {
+    fun identifiserKontrollerpunkter(hendelse: Doedshendelse): List<DoedshendelseKontrollpunkt> {
+        val kontrollpunkter = mutableListOf<DoedshendelseKontrollpunkt>()
```

Comment:
Hvorfor ikke bare `mutableListOf<DoedshendelseKontrollpunkt?>()` og `return kontrollpunkter.mapNotNull { it }`

Reply:
Trenger jo egentlig ikke bruke en mutable liste, så endret til:
```
return listOfNotNull(
    kontrollerAvdoedDoedsdato(avdoed),
    kontrollerKryssendeYtelse(hendelse),
    kontrollerDNummer(avdoed),
    kontrollerUtvandring(avdoed),
)
```